.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,6 +30,6 @@ Fixes #<ISSUE ID>
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] My code has been rebased from devel before I start
-- [ ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
+- [ ] I have read the [**CONTRIBUTING**](https://avd.arista.com/devel/docs/contribution/overview.html) document.
 - [ ] My change requires a change to the documentation and documentation have been updated accordingly. (check the box if not applicable)
 - [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)
```

Comment:
is this correct?

Reply:
Yes, this is historical as we didn't maintain a separate contribution guide for ansible-cvp.